test/BaselineOutput/Common/EntryPoints/core_manifest.json

```diff
@@ -2727,6 +2727,114 @@
           "SortOrder": 7.0,
           "IsNullable": false,
           "Default": null
+        },
+        {
+          "Name": "IgnoreColumn",
+          "Type": {
+            "Kind": "Array",
+            "ItemType": "String"
+          },
+          "Desc": "Column(s) to use as purpose 'Ignore'",
+          "Required": false,
+          "SortOrder": 8.0,
+          "IsNullable": false,
+          "Default": null
+        },
+        {
+          "Name": "NameColumn",
+          "Type": {
+            "Kind": "Array",
+            "ItemType": "String"
+          },
+          "Desc": "Column(s) to use as purpose 'Name'",
+          "Required": false,
+          "SortOrder": 9.0,
+          "IsNullable": false,
+          "Default": null
+        },
+        {
+          "Name": "LabelColumn",
+          "Type": {
+            "Kind": "Array",
+            "ItemType": "String"
+          },
+          "Desc": "Column(s) to use as purpose 'Label'",
+          "Required": false,
+          "SortOrder": 10.0,
+          "IsNullable": false,
+          "Default": null
+        },
+        {
+          "Name": "NumericFeatureColumn",
```

Comment:
>NumericFeatureColumn [](start = 19, length = 20)

I'd keep Column plural: NumericFeatureColumns

Reply:
Using plural form as suggested

---
In reply to: [199593350](https://github.com/dotnet/machinelearning/pull/461#discussion_r199593350) [](ancestors = 199593350)